### PR TITLE
Add "suicide" and "suicidal" and others

### PR DESCRIPTION
**Pull request by llunariis**
Adds some more profane words.

### CODE REVIEW

**Review by meltfrommeltland — APPROVED** [review]

**Comment by meltfrommeltland** [comment]
oops

**Comment by llunariis** [comment]
> oops

Hello.

File: better_profanity/profanity_wordlist.txt [hunk 1]
```diff
@@ -833,3 +833,29 @@ xx
...
+homophobia
+NSFW
+not safe for work
+kys
+kill yourself
+frick
+butt
+racist
+racisim
+farted
```
Comment (meltfrommeltland) [inline better_profanity/profanity_wordlist.txt]: Remove this

File: better_profanity/profanity_wordlist.txt [hunk 2]
```diff
@@ -833,3 +833,29 @@ xx
...
+farting
```
Comment (meltfrommeltland) [inline better_profanity/profanity_wordlist.txt]: Remove this

File: better_profanity/profanity_wordlist.txt [hunk 3]
```diff
@@ -833,3 +833,29 @@ xx
...
+wee
```
Comment (meltfrommeltland) [inline better_profanity/profanity_wordlist.txt]: Remove this

File: better_profanity/profanity_wordlist.txt (hunk 1 above)
Comment (meltfrommeltland) [reply better_profanity/profanity_wordlist.txt]: Makes no sense to be on there

File: better_profanity/profanity_wordlist.txt (hunk 2 above)
Comment (meltfrommeltland) [reply better_profanity/profanity_wordlist.txt]: Same reason as farted

File: better_profanity/profanity_wordlist.txt (hunk 3 above)
Comment (meltfrommeltland) [reply better_profanity/profanity_wordlist.txt]: Wee isn't profanity

File: better_profanity/profanity_wordlist.txt [hunk 4]
```diff
@@ -833,3 +833,29 @@ xx
...
+pervert
+simp
+simps
+devil
```
Comment (meltfrommeltland) [inline better_profanity/profanity_wordlist.txt]: Remove this, it's not necessary

File: better_profanity/profanity_wordlist.txt [hunk 5]
```diff
@@ -833,3 +833,29 @@ xx
...
+satan
```
Comment (meltfrommeltland) [inline better_profanity/profanity_wordlist.txt]: Remove this, same reason as devil.

File: better_profanity/profanity_wordlist.txt [hunk 6]
```diff
@@ -833,3 +833,29 @@ xx
...
+666
```
Comment (meltfrommeltland) [inline better_profanity/profanity_wordlist.txt]: Remove this, also same reason as devil

File: better_profanity/profanity_wordlist.txt [hunk 7]
```diff
@@ -833,3 +833,29 @@ xx
...
+satanist
```
Comment (meltfrommeltland) [inline better_profanity/profanity_wordlist.txt]: Remove this, no need for religions to be blacklisted

File: better_profanity/profanity_wordlist.txt [hunk 8]
```diff
@@ -833,3 +833,29 @@ xx
...
+gimp
+hobo
```
Comment (meltfrommeltland) [inline better_profanity/profanity_wordlist.txt]: Remove this, not needed.

File: better_profanity/profanity_wordlist.txt [hunk 9]
```diff
@@ -833,3 +833,29 @@ xx
...
+satanists
```
Comment (meltfrommeltland) [inline better_profanity/profanity_wordlist.txt]: Remove this, same reason as satanist.

**Review by meltfrommeltland — CHANGES_REQUESTED** [review]
Remove racist, racism, farted, farting, wee, devil, satan, 666, hobo, butt, and satanists.

**Pull request closed without merging**